### DISCUSSION TITLE
241127_수

### DISCUSSION THREAD

**Discussion by Soy17** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#61] [TIL]
- [x] [ALL] 민영쓰의 생일 축하해주기🎉
- [ ] **TODO**

**Comment by Yoon0717** [comment]
- [x] 2048 성능 올리는 방법 모색 -> 시간이 너무 오래 걸려서 중단
- [x] albumentation을 활용한 전처리
- [x] mixup 재도전

**Comment by wonjeongjeong** [comment]
- [x] 퀴즈 8~10
- [ ] 소프트 보팅 앙상블 구현 -> ai stage 게시판 활용

**Comment by sejongmin** [comment]
- [ ] wandb validation gt & pred 시각화 해보기
- [ ] wandb 모델 올리는 것 없애기

**Comment by minrongtic** [comment]
**TO DO**

**Comment by june21a** [comment]
- [x] soft voting 코드 분석해보기
- [x] mmseg에서의 적용을 탐구..
- [ ] 아니 batch size 2로 안하면 batchnorm이 오류나서 다른 backbone 찾아보기..
